maud/tests/warnings/keyword_names_without_@.stderr

```diff
@@ -0,0 +1,7 @@
+error: found keyword `if`
+ --> $DIR/keyword_names_without_@.rs:9:9
```

Comment:
Nit: is it good style to have an `@` symbol in a test filename?

Not sure what the convention is for trybuild test cases.

Reply:
Well, I don't see `@` as a case of bad naming. If it was something obscure, sure, but the symbol is ASCII and can be typed with literally every keyboard. Anyway, if you don't happy, I could rename it to `keyword_names_without_at_sign` or something.

`trybuild` has no limitations for filenames AFAIK. You could even have an emoji in it if it was a wicked desire of yours...

Reply:
Sounds good to me. My question was more whether this is *idiomatic*, rather than whether it works (the Travis build passes so I know it does). But it sounds like you've thought about this so I'm happy to resolve.